config/encryption.go

```diff
@@ -0,0 +1,133 @@
+package config
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+func GetKey(path string) (key string, err error) {
+	key, err = readKeyFromFile(path)
+	if err != nil {
+		key, err = generateKey(path)
+		if err != nil {
+			return "", fmt.Errorf("config: could not read or generate key: %w", err)
+		}
+	}
+
+	return key, nil
+}
+
+// readKeyFromFile reads key from file it it exists.
+func readKeyFromFile(path string) (key string, err error) {
+	uint8key, err := ioutil.ReadFile(path)
+	if err != nil {
+		return "", fmt.Errorf("application: could not read key file: %w", err)
+	}
+
+	return string(uint8key), nil
+}
+
+// generateKey creates new .txt file and generates random key.
+func generateKey(path string) (newKey string, err error) {
+	f, err := os.Create(path)
```

Comment:
One last thing. Please add chmod 600 to the key, so it's not like anybody can ready it and decrypt it on their own.